src/dex/verified/constants.ts

```diff
@@ -1,10 +1,62 @@
-export const MIN_USD_LIQUIDITY_TO_FETCH = 100n;
-
-// Let's take this trade as an example:
-// https://dashboard.tenderly.co/paraswap/paraswap/fork/e4c81946-fd6e-4299-b35c-c47b775e3c05/simulation/8839462c-6239-4ae1-9ed0-8013f89b41de/gas-usage
+import VAULTABI from '../../abi/verified/vault.json';
+import PRIMARYISSUE from '../../abi/verified/PrimaryIssuePool.json';
+import SECONDARYISSUE from '../../abi/verified/SecondaryIssuePool.json';
+import { Interface } from 'ethers/lib/utils';
 
-// 271719 - 57856 - 79098 - 51041 = 83724 ~ 84k
-export const STABLE_GAS_COST = 84_000;
-
-// I see three pools used in trade: (57856 + 79098 + 51041) / 3 = 62665 ~ 63k
-export const VARIABLE_GAS_COST_PER_CYCLE = 63_000;
+export const MIN_USD_LIQUIDITY_TO_FETCH = 100n;
+export const fetchAllPools = `query ($count: Int)    {
```

Comment:
@Mohzcrea8me should a function be in constants ?

Reply:
It's a function in string form used as query for subgraph

I will move it back to verified-pool